src/official_interface.py

```diff
@@ -6,6 +6,19 @@
 import requests
 import cv2
 
+
+class ResponseError(Exception):
+    """レスポンスエラー用の例外."""
+
+    def __init__(self, message):
```

Comment:
```suggestion
    def __init__(self, message: str):
```
引数のところに型指定がないです。他のところもお願いします